src/browser/ui/ReactMount.js

```diff
@@ -451,12 +457,13 @@ var ReactMount = {
    * perform an update on it and only mutate the DOM as necessary to reflect the
    * latest React component.
    *
+   * @param {ReactElement} parentComponent The conceptual parent of this render tree.
    * @param {ReactElement} nextElement Component element to render.
    * @param {DOMElement} container DOM element to render into.
    * @param {?function} callback function triggered on completion
    * @return {ReactComponent} Component instance rendered in `container`.
    */
-  render: function(nextElement, container, callback) {
+  renderSubtreeIntoContainer: function(parentComponent, nextElement, container, callback) {
```

Comment:
Yeah, I think Ben was also in favor of putting it on the component in order to preserve argument order.  I sort of like it too, because I see it like "asking this component to render a child" which seems better than rendering a child on behalf of the component.  But since I don't think this should be public API yet (I would have underscored it), I think addons is the right place for it, as per our discussion.  Moved it there.